pallets/asset/src/benchmarking.rs

```diff
@@ -122,35 +151,27 @@ benchmarks! {
 
         let caller = UserBuilder::<T>::default().generate_did().build("caller");
         // Generate a ticker of length `t`.
-        let ticker = Ticker::try_from(vec![b'A'; TICKER_LEN].as_slice()).unwrap(); }: _(caller.origin, ticker.clone()) verify {
+        let ticker = Ticker::try_from(vec![b'A'; TICKER_LEN].as_slice()).unwrap();
```

Comment:
You can use `Ticker::repeating()` or the `generate_ticker` function at places where you need a ticker.